sofa-boot-project/sofa-boot/pom.xml

```diff
@@ -45,6 +45,12 @@
             <artifactId>logback-classic</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
why add this dependency？

Reply:
The ci run fail, please check it.

Reply:
- removed the dependency it was a mistake.
- PMD fails are resolved pipeline should not break now.

Reply:
please run mvn compile to updade code format before you submit pr